sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -261,6 +261,8 @@ object FunctionRegistry {
     expression[VarianceSamp]("var_samp"),
     expression[CollectList]("collect_list"),
     expression[CollectSet]("collect_set"),
+    expression[CountMinSketchAgg]("count_min_sketch"),
+    expression[CountMinSketchAgg]("cmsketch"),
```

Comment:
actually i take my word back. let's add only count_min_sketch. I don't think it's worth having an alias given this is sketch is difficult to consume (returning some binary)